src/uwtools/tests/drivers/test_chgres_cube.py

```diff
@@ -97,8 +98,10 @@ def test_ChgresCube():
         "_validate",
         "_write_runscript",
         "run",
-    ]:
-        assert getattr(ChgresCube, method) is getattr(Driver, method)
+    ],
+)
+def test_ChgresCube(method):
```

Comment:
Is my assumption correct that this could help even more with the ability to run tasks more quickly in parallel? I mean maybe not this test explicitly since it looks like it might be fairly inexpensive, but in general.

Reply:
I hadn't thought of it but, yes, I think so: This test would previously have had to be run by a single worker, and now each of its cases can be executed concurrently.